src/modules/item/selectors.ts

```diff
@@ -58,6 +58,13 @@ export const getWalletOrphanItems = createSelector<RootState, Item[], Item[]>(ge
   items.filter(item => item.collectionId === undefined)
 )
 
+export const getPaginatedWalletOrphanItems = (state: RootState, address: string, pageSize?: number) => {
```

Comment:
Would you mind adding tests for this selector?

Reply:
done!